### ISSUE TITLE
Do not force CodeLens (or explain it well)

### ISSUE THREAD

**Issue by langpavel** [Hotell/typescript-lib-starter#145]
Hi
you are forcing VSCode users to:
```
  "typescript.referencesCodeLens.enabled": true,
  "javascript.referencesCodeLens.enabled": true,
```

As I can imagine — it can be very powerful tool. But it doesn't work for me and I *must* explicitly disable it.

Will be much better to instruct people how to turn this on instead of letting us (*scratching on the head*) what is that `0 references` disturbing line…

**Comment by Hotell** [comment]
Hmm, purpose of this package is not to explain code editor configuration and what it does. 
I'll provide option in new init script, which will ask user if they wanna include vscode recommended settings for this library. that's all what I can do and it's right IMHO, if it helps :)

**Comment by langpavel** [comment]
That CodeLens are explicitly enabled by your starter.I suggest no force, you can kindly recommend in

**Comment by Hotell** [comment]
Closed by #143
